t/08-autotest.t

```diff
@@ -449,9 +449,14 @@ subtest make_snapshot => sub {
 };
 
 subtest loadtestdir => sub {
-    throws_ok {
-        autotest::loadtestdir('test-snapshot');
-    } qr/does not exist!/, 'test died as expected';
+    use lib qw(t/data/tests);
+    $bmwqemu::vars{CASEDIR} = 't/data/tests';
+    stderr_like {
+        like warning {
+            autotest::loadtestdir('tests');
+        }, qr/ARRAY/, 'script found';
+    } qr/scheduling/, 'loadscript success';
+
```

Comment:
Just to be sure that loadtestdir have load the directory we should check if the package variable `tests` contains one of the script in the directory, for example `boot.pm` the loadtest subroutine should create a `tests-boot` element
```suggestion
ok exists $autotest::tests{'tests-boot'}, 'boot.pm loaded';
```

If you want to check the content of the `tests` variable there is [`Data::Dumper`](https://metacpan.org/pod/Data::Dumper) module that can be use to translate data structures into string.